tests/browserstack/specs/valuesMainTest.js

```diff
@@ -0,0 +1,218 @@
+const { simpleClick, simpleTextInput, scrollIntoViewSimple } = require('../utils');
+
+const ANDROID_CONTEXT = 'WEBVIEW_org.wevote.cordova';
+const IOS_CONTEXT = 'WEBVIEW_1';
+const PAUSE_DURATION_MICROSECONDS = 250;
+
+describe('Basic cross-platform We Vote test',  () => {
+  it('should load the app so we can run tests', async () => {
+    const { isCordovaFromAppStore, isMobileScreenSize } = driver.config.capabilities;
+    const isDesktopScreenSize = !isMobileScreenSize;
+    const xssTest = '<script>alert(1)</script>';
+    const publicFigureOrOrganizationFollowSelector = '[id^=positionItemFollowToggleFollow-undefined-wv02org]';
+    const publicFigureOrOrganizationDropDownSelector = '[id^=positionItemFollowToggleDropdown-undefined-wv02org]';
+    const publicFigureOrOrganizationUnfollowSelector = '[id^=positionItemFollowToggleUnfollow-undefined-wv02org]';
+    const publicFigureOrOrganizationIgnoreSelector = '[id^=positionItemFollowToggleIgnore-undefined-wv02org]';
+    const publicFigureOrOrganizationUnignoreSelector = '[id^=positionItemFollowToggleStopIgnoring-undefined-wv02org]';
+    const organizationSection = '#mainContainer:nth-child(4)';
+
+    if (isCordovaFromAppStore) {
+    // ///////////////////////////////
+    // For the apps downloadable from either the Apple App Store or Android Play Store,
+    // click through the onboarding screens
+      await browser.pause(PAUSE_DURATION_MICROSECONDS * 2);
+      const contexts = await driver.getContexts();
+      const context = contexts.includes(ANDROID_CONTEXT) ? ANDROID_CONTEXT : IOS_CONTEXT;
+      await driver.switchContext(context);
+      await browser.pause(PAUSE_DURATION_MICROSECONDS);
+      const firstNextButton = await $('div[data-index="0"] .intro-story__btn--bottom');
+      await firstNextButton.click();
+      await browser.pause(PAUSE_DURATION_MICROSECONDS);
+      const secondNextButton = await $('div[data-index="1"] .intro-story__btn--bottom');
+      await secondNextButton.click();
+      await browser.pause(PAUSE_DURATION_MICROSECONDS);
+      const thirdNextButton = await $('div[data-index="2"] .intro-story__btn--bottom');
+      await thirdNextButton.click();
+      await browser.pause(PAUSE_DURATION_MICROSECONDS);
+    } else {
+    // ///////////////////////////////
+    // For the website version, open our quality testing site
+      await browser.url('http://localhost:3000/values');
```

Comment:
Hi @4D617278, could you create a variable in tests/browserstack/browserstack.config-template.js for `WEB_APP_ROOT_URL: 'https://quality.wevote.us',` that you can change on your local machine in tests/browserstack/browserstack.config.js to `WEB_APP_ROOT_URL: 'http://localhost:3000',`? (Note, we use the variable name `WEB_APP_ROOT_URL` in WeVoteServer too.)